### PR TITLE
fix response data

### PR DESCRIPTION
**Pull request by shotasilagadzetaal**

### CODE REVIEW

File: pkg/api/handler/default.go [hunk 1]
```diff
@@ -187,13 +186,16 @@ func (m ArcDefaultHandler) GETTransactionStatus(ctx echo.Context, id string) err
...
+	emptyString := ""
+
 	return ctx.JSON(http.StatusOK, api.TransactionStatus{
 		BlockHash:   &tx.BlockHash,
 		BlockHeight: &tx.BlockHeight,
 		TxStatus:    &tx.Status,
 		Timestamp:   m.now(),
 		Txid:        tx.TxID,
 		MerklePath:  &tx.MerklePath,
+		ExtraInfo:   &emptyString,
```
Comment (boecklim) [inline pkg/api/handler/default.go:198]: You could use this instead of creating that empty string variable `PtrTo(""),`

File: pkg/api/arc.yml [hunk 2]
```diff
@@ -551,21 +538,8 @@ components:
     TransactionSubmitStatus:
```
Comment (boecklim) [inline pkg/api/arc.yml:538]: I think you can now remove the whole `TransactionSubmitStatus` schema as it does not contain anything anymore

File: pkg/api/status.go [hunk 3]
```diff
@@ -25,8 +25,10 @@ const (
 )
 
 func NewErrorFields(status StatusCode, extraInfo string) *ErrorFields {
+	emptyString := ""
 	errFields := ErrorFields{
-		Status: int(status),
+		Status:    int(status),
+		ExtraInfo: &emptyString,
```
Comment (boecklim) [inline pkg/api/status.go:31]: You could use this instead of creating that empty string variable PtrTo(""),

Comment (shotasilagadzetaal) [reply pkg/api/status.go:31]: yeah, I tried that but `PtrTo` is undefined there and didn't want to copy function.

File: pkg/api/handler/default.go (hunk 1 above)
Comment (shotasilagadzetaal) [reply pkg/api/handler/default.go:198]: same here, `PtrTo` is undefined.

Comment (boecklim) [reply pkg/api/handler/default.go:198]: Have you checked? 
It is defined

```
	return ctx.JSON(http.StatusOK, api.TransactionStatus{
		BlockHash:   &tx.BlockHash,
		BlockHeight: &tx.BlockHeight,
		TxStatus:    &tx.Status,
		Timestamp:   m.now(),
		Txid:        tx.TxID,
		MerklePath:  &tx.MerklePath,
		ExtraInfo:   PtrTo(""),
	})
```

Comment (shotasilagadzetaal) [reply pkg/api/handler/default.go]: Oh, it should work here

**Pull request merged**